torchaudio/datasets/dr_vctk.py

```diff
@@ -0,0 +1,109 @@
+from pathlib import Path
+from typing import Dict, Tuple, Union
+
+from torch import Tensor
+from torch.utils.data import Dataset
+
+import torchaudio
+from torchaudio.datasets.utils import (
+    download_url,
+    extract_archive,
+)
+
+
+URL = "https://datashare.ed.ac.uk/bitstream/handle/10283/3038/DR-VCTK.zip"
```

Comment:
Can prefix this with underscode? URL might change anytime, so it's safer to let client code not use it.